SwiftUIHomeworks/SwiftUIHomeworks/Screens/CartView.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  CartView.swift
+//  SwiftUIHomeworks
+//
+//  Created by Aleksandr Dorofeev on 21.01.2023.
+//
+
+import SwiftUI
+
+/// Cart view
+struct CartView: View {
+  // MARK: - Private Constants
+  
+  private enum Constants {
+    static let emptyCartTitle = "Cart is empty"
+    static let navigationTitle = "Cart"
+    static let payButtonTitle = "Pay"
+    static let infoAboutOrderTitle = "Info about order"
+    static let confirmChoiceMessage = "Confirm your choice\n"
+    static let opacityZeroValue: CGFloat = 0
+    static let opacityOneValue: CGFloat = 1
+    static let weightButtonValue: CGFloat = 100
+    static let heightButtonValue: CGFloat = 50
+    static let rubTitle = "RUB"
+    static let daysTitle = "days"
+  }
+  
+  // MARK: - Body
+  
+  var body: some View {
+    NavigationView {
+      ZStack {
+        Text(Constants.emptyCartTitle)
+        buySectionView
+          .opacity(purchaseViewModel.isBuying ? Constants.opacityOneValue : Constants.opacityZeroValue)
+      }
+      .navigationTitle(Constants.navigationTitle)
+    }
+  }
+  
+  // MARK: - Private properties
+  
+  @EnvironmentObject private var purchaseViewModel: PurchaseViewModel
+  @State private var isConfirmationAlertShown = false {
+    didSet {
+      purchaseViewModel.isEnabled = true
+    }
+  }
+  
+  private var buySectionView: some View {
+    Form {
+      Section {
+        HStack {
+          Text("\(purchaseViewModel.limitationValue) \(Constants.daysTitle)")
+          Spacer()
+          Text("\(purchaseViewModel.priceValue) \(Constants.rubTitle)")
+        }
```

Comment:
```suggestion
        HStack {
          Text("\(purchaseViewModel.limitationValue) \(Constants.daysTitle)")
          Spacer()
          Text("\(purchaseViewModel.priceValue) \(Constants.rubTitle)")
        }
```
можно вынести в отдельное свойство